### ISSUE TITLE
Buttons behave opposite of what they should

### ISSUE THREAD

**Issue by iAlborz** [romashamin/sort-me-sketch#4]
A1 > Z9 sorts like Z0 > A1 and vice versa

**Comment by romashamin** [comment]
Default sort order for layers is ‘newest on top’. ⌃⇧O produces natural (for layers) order:
- Layer 3
- Layer 2
- Layer 1

**Comment by iAlborz** [comment]
I understand how Sketch has an unintuitive approach to the sorting, but the whole purpose of this plug-in is to make it intuitive. As a user I don't really care how sketch sorts them, but I do care for example when I select 1>9 it would go
- 1
- 2
- 3

and not the other way around. maybe reversing the functions (or just simply changing the names of the functions) would make it more intuitive for the user?

**Comment by romashamin** [comment]
Hey, we’ve already discussed it.

You have two options:
1. Remember shortcut that produces the order you want. In your case it’s Ctrl+Shift+P.
2. Fork the repo, change the code in the way you want.

**Comment by ahuk** [comment]
Agree with @iAlborz. @romashamin Do users remember which artboard was created prior to or after that artboard X?

IMHO, the classical divide: Does technology determine how users have to use a tool or does user behaviour determine the implementation logic? 

I assume that "newest on top" came from PS way back in the days when a new (duplicated) layer **had** to be on top or one would not see it. This behaviour does not hold with Sketch as artboards are not placed on top of each other by Sketch (IMHO one of the main reasons why Sketch got so popular: nicely placed artboards on a big canvas, easy to view, easy to manage).

Bottom line: I assume that 95% of Sketch users will wonder why the plugin does "Sort Z9 > A1", does actually do "A1 > Z9".

Quick fix to remove confusion: rename to "Newest on top" / "Oldest on top" (but then do users remember what is _new_ and what is _old_?)
